riva/proto/riva_asr.proto

```diff
@@ -111,6 +111,15 @@ message StreamingRecognizeRequest {
   RequestId id = 100;
 }
 
+// Provides information on all end of utterance related values
+message EOUConfig {
```

Comment:
Also, should we rename the proto message to `EndpointingConfig`? Thoughts?

Reply:
Let's keep it `EOUConfig` .

Reply:
As suggested, renamed all parameters as suggested. Current implementation uses a custom message

Reply:
LGTM!

Reply:
agree with `EndpointingConfig` as name of message